vendor/CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ add_compile_definitions(_FILE_OFFSET_BITS=64 _LARGEFILE64_SOURCE)
 set(android-vendored
 	avb
 	adb
-	base
```

Comment:
I don't think you want to remove `base` submodule here.

Reply:
Oh wait, does it mean that `libandroidfw` was the only library from `base` used?

Reply:
Correct, that submodule existed solely in order to provide libandroidfw -- I originally tested this via `rm -rf vendor/base` and everything still built.